packages/dashboard-frontend/src/Layout/Header/Tools/UserMenu/__tests__/index.spec.tsx

```diff
@@ -44,27 +44,14 @@ describe('User Menu', () => {
   global.open = jest.fn();
 
   const email = 'johndoe@example.com';
-  const name = 'John Doe';
-  const store = createStore(name, email);
+  const username = 'John Doe';
+  const store = createStore(username, email);
   const history = createHashHistory();
-  const user = {
-    id: 'test-id',
-    name: name,
-    email: email,
-    links: [],
-  };
   const branding = selectBranding(store.getState());
-  const userProfile = selectUserProfile(store.getState());
```

Comment:
Could you remove all unused imports after your changes too?

Reply:
done